db/migrate/20181001141138_ignore_taxonomies_for_audit_filters.rb

```diff
@@ -1,6 +1,6 @@
 class IgnoreTaxonomiesForAuditFilters < ActiveRecord::Migration[5.2]
```

Comment:
Should we just remove this file so we don't execute things twice?

Reply:
We could just remove it, if Rails has no problem with that, otherwise we could just make this migration a no-op and simply return.

Reply:
Rails don't have problem, but devs searching what the migration was when debugging something will not be happy. If there is some performance concern or something, I'd prefer to keep it blank, so it's still in git.